### PR TITLE
Funneljoin post

### PR DESCRIPTION
**Pull request by derekbeaton**
# Summary

Provide a summary of your blog post here.

# Checklist (Writer)

- [X] I can successfully knit the blog post Rmd file.

- [X] Figures, graphs, tables render properly in the blog post Rmd file.

- [X] Abbreviations are expanded/explained.

- [X] The blog post Rmd file header contains the post title, post description, author information, and categories.

- [X] The blog post content (Rmd file, figures, tables, etc.) does not contain any PHI.

# Checklist (AA Blog Team Reviewers)

- [ ] I have updated the blog's draft status from 'TRUE' to 'FALSE'

- [ ] I have updated the date on the blog post to reflect the release date

- [ ] I have renamed the blog post directory so the date matches the release date

- [ ] I have re-rendered the blog post and all required files are present

### CODE REVIEW

**Comment by derekbeaton** [comment]
I've already made a small edit because two sentences in a row started with "So at that point [...]" and that's just bad writing!

File: _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd [hunk 1]
```diff
@@ -0,0 +1,289 @@
+---
+title: "Putting the fun in `funneljoin`"
```
Comment (chloepouprom) [inline _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd:2]: 🥳

File: _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+  mode = "inner",
+  type = "first-firstafter"
+)
+
+rmarkdown::paged_table(join_attempt_two)
+
+
+```
+
+The above shows the whole table but let's take a closer look at something...
```
Comment (chloepouprom) [inline _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd:211]: This shows up as empty for me 🤔 
image

File: _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+  kableExtra::kbl() %>%
+  kableExtra::kable_styling() 
+
+
+```
+
+
+# Conclusions
+
+The first conclusion is that `funneljoin` is awesome and the second conclusion is that I'm lazy. It is also a safe assumption to conclude that I've used an excessive amount of unnecessary footnotes^[not enough footnotes and they are totally necessary].
```
Comment (chloepouprom) [inline _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd:281]: 👏 more 👏 footnotes 👏 !!

File: _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+
+ADT_EVENTS %>%
+  slice( c(15, 16, 17) ) %>%
+  kableExtra::kbl() %>%
+  kableExtra::kable_styling() 
+
+
+```
+
+A fun note before we move on: When you look at these data you'll see some `NA` in there. Those `NA` are `R`s `NA` which is effectively missing data. In this case `NA` is absolutely not NA which is sodium (sodium is not a service). You should really take a look at some of the ["fun"](https://lks-chart.github.io/blog/posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/) with `NA` we've had.
```
Comment (meggie-d) [inline _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd:116]: 😆

File: _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd (hunk 2 above)
Comment (derekbeaton) [reply _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd:211]: which version of R are you using? If it's 4.0.2 I think it'll be OK. If it's 3.6.3, then I believe it requires `stringsAsFactors = FALSE`. Which I should probably make explicit anyways...

**Review by meggie-d — APPROVED** [review]
A *fun* read!! 👍

File: _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd (hunk 2 above)
Comment (chloepouprom) [reply _posts/2022-08-11-putting-the-fun-in-funneljoin/putting-the-fun-in-funneljoin.Rmd:215]: It works for me now!

**Review by chloepouprom — APPROVED** [review]

**Pull request merged**